ted_sws/data_manager/entrypoints/cli/cmd_export_notices_from_mongodb.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+import click
+from pathlib import Path
+
+from pymongo import MongoClient
+
+from ted_sws import config
+from ted_sws.core.adapters.cmd_runner import CmdRunner as BaseCmdRunner
+from ted_sws.data_manager.services.export_notice_from_mongodb import export_notice_by_id
+
+CMD_NAME = "CMD_EXPORT_NOTICES_FROM_MONGODB"
+
+"""
+USAGE:
+# component_detector --help
+"""
+
+
+class CmdRunner(BaseCmdRunner):
+
+    def __init__(self, notice_ids: List[str] = None, output_folder: str = None, mongodb_client: MongoClient = None):
+        super().__init__(name=CMD_NAME)
+        self.notice_ids = self._init_list_input_opts(notice_ids)
+        self.output_folder = output_folder
+        self.mongodb_client = mongodb_client
+
+    def run_cmd(self):
+        if self.notice_ids:
+            for notice_id in self.notice_ids:
+                is_saved, saved_path = export_notice_by_id(notice_id, self.output_folder,
+                                                           mongodb_client=self.mongodb_client)
+                if is_saved:
+                    self.log(f"Notice {notice_id} saved in {saved_path}.")
+                else:
+                    self.log(f"Notice {notice_id} is not saved.")
+        else:
+            self.log("List with notices is empty.")
+
+        return self.run_cmd_result()
+
+
+def run(notice_id=None,
+        output_folder: str = None,
+        mongodb_client: MongoClient = None):
+    if not mongodb_client:
+        mongodb_client = MongoClient(config.MONGO_DB_AUTH_URL)
+
+    cmd = CmdRunner(
+        notice_ids=list(notice_id or []),
+        output_folder=output_folder,
+        mongodb_client=mongodb_client
+    )
+    cmd.run()
+
+
+@click.command()
+@click.option('-n', '--notice-id', required=True, multiple=True)
+@click.option('-o', '--output-folder', required=False, default=".")
```

Comment:
could we make the CLI to take as optional MONGO_DB_AUTH_URL?

Reply:
export_notices_from_mongodb --output-folder=./tmp --notice-id=000163-2021 --mongodb-auth-url="mongodb://user:pass@localhost:27017/"

if mongo-url not provided the .env value will be used